source/renderer/app/stores/AppStore.js

```diff
@@ -13,39 +12,38 @@ import {
 } from '../ipc/control-ui-parts';
 import { getGPUStatusChannel } from '../ipc/get-gpu-status.ipc';
 import { generateFileNameWithTimestamp } from '../../../common/utils/files';
-
 import type { GpuStatus } from '../types/gpuStatus';
 
 export default class AppStore extends Store {
   @observable error: ?LocalizableError = null;
-  @observable isAboutDialogOpen = false;
-  @observable isDaedalusDiagnosticsDialogOpen = false;
   @observable isDownloadNotificationVisible = false;
   @observable gpuStatus: ?GpuStatus = null;
   @observable numberOfEpochsConsolidated: number = 0;
   @observable previousRoute: string = ROUTES.ROOT;
+  @observable activeDialog: string = '';
```

Comment:
@DeeJayElly can we please have a special `Dialog` (or some better name) type and use null for default when there is no active dialog?

Reply:
@nikolaglumac done

Reply:
@DeeJayElly I am afraid you did not understand me here - I did not want to add `current` and type that - I wanted the `ActiveScreenType` to not be a string but rather a list of possible dialogs such as that this line reads:
```
@observable activeDialog: ActiveDialogName = null;
```
...and then `ActiveDialogName` is defined like:
```
export type ActiveDialogName =
  | 'about'
  | 'adaRedemption'
  | 'blockConsolidation'
  | 'daedalusDiagnostics'
  | null;
```

Reply:
yep i changed that part now. Its more cleaner and packed in external type & const.